src/site/content/en/progressive-web-apps/add-manifest/index.md

```diff
@@ -0,0 +1,231 @@
+the app name, icons it should use, and the URL that should be opened when the
+app is launched.
+
+Manifest files are supported in Chrome, Edge, Firefox, UC Browser, Opera,
+the Samsung browser, and Safari partial support.
+
+## Create the manifest.webmanifest file
+
+The manifest file can have any name, but is commonly named
+`manifest.webmanifest` and served from the root (your website's top level
+directory).
+
+```json
+{
+  "short_name": "Weather",
+  "name": "Weather: Do I need an umbrella?",
+  "description": "Weather forecast information",
+  "icons": [
+    {
+      "src": "/images/icons-192.png",
+      "type": "image/png",
+      "sizes": "192x192"
+    },
+    {
+      "src": "/images/icons-512.png",
+      "type": "image/png",
+      "sizes": "512x512"
+    }
+  ],
+  "start_url": "/?source=pwa",
+  "background_color": "#3367D6",
+  "display": "standalone",
+  "scope": "/",
+  "theme_color": "#3367D6"
+}
+```
+
+### Key manifest properties
+
+#### `short_name` and/or `name` {: #name }
+
+You must provide at least the `short_name` or `name` property. If both are
+provided, `short_name` is used on the user's home screen, launcher, or other
+places where space may be limited. `name` is used when the app is installed.
+
+#### `icons` {: #icons }
+
+When a user installs your PWA, you can define a set of icons for the browser
+to use. These icons are used in places like the home screen, app launcher,
+task switcher, splash screen, etc.
+
+The `icons` property is an array of image objects. Each object should
+include the `src`, a `sizes` property, and the `type` of image. To use
+[maskable icons](/maskable-icon/), sometimes referred to as adaptive
+icons on Android, you'll also need to add `"purpose": "any maskable"` to the
+`icon` property.
+
+For Chrome, you must provide at least a 192x192 pixel icon, and a 512x512
+pixel icon. If only those two icon sizes are provided, Chrome will
+automatically scale the icons to fit the device. If you'd prefer to scale your
+own icons, and adjust them for pixel-perfection, provide icons in increments
+of 48dp.
+
+#### `start_url` {: #start-url }
+
+The `start_url` is required and tells the browser where your application
+should start when it is launched, and prevents the app from starting on
+whatever page the user was on when they added your app to their home screen.
+
+Your `start_url` should direct the user straight into your app, rather than
+a product landing page. Think about what the user will want to do once
+they open your app, and place them there.
+
+#### `background_color` {: #background-color }
+
+The `background_color` property is used on the splash screen when the
+application is first launched on mobile.
+
+#### `display` {: #display }
+
+You can customize what browser UI is shown when your app is launched. For
+example, you can hide the address bar and browser chrome. Or games may want
+to go completely full screen.
```

Comment:
"Or games may want to go completely full screen." to "Games can even be made to launch fullscreen." 

Issues:
* Starting sentence with 'or'. 
* Anthropomorphizing the game.